baseplate/context/memcache/__init__.py

```diff
@@ -31,6 +32,17 @@ def pool_from_config(app_config, prefix="memcache.", **kwargs):
         socket connected to memcache. Defaults to the underlying socket default
         timeout.
 
+    :param dict app_config: the config dictionary
+    :param str prefix: prefix for config keys
+    :param func serializer: function to serialize values to strings suitable
+        for being stored in memcached. An example is
+        ``baseplate.context.memcache.lib.make_memcache_serializer``
```

Comment:
I think you can link these functions with `:py:func:...` for a little more sphinx juice.